msteams-platform/bots/how-to/conversations/notification-bot-in-teams.md

```diff
@@ -0,0 +1,579 @@
+---
+title: Notification bot in Teams
+author: surbhigupta
+description: Learn how a notification bot works in Teams, and to customize notification behavior.
+ms.topic: conceptual
+ms.author: v-amprasad
+ms.localizationpriority: high
+---
+
+# Notification bot in Teams
+
+Microsoft Teams Toolkit enables you to build applications that capture events and send them as notifications to a personal, group chat, or a channel in Teams. You can send notifications as plain text or Adaptive Cards. The notification bot template creates an app that sends a message to Teams with Adaptive Cards triggered by HTTP post request.
+
+The app template is built using the TeamsFx SDK, which provides a simple set of functions over Microsoft Bot Framework to implement your requirement. For example, in a scenario where a travel agency builds an app in Teams for their customers to keep them up-to-date with the weather forecast. In the following diagram you can see a Teams app that sends notification to the travelers about the destination weather forecast:
+
+:::image type="content" source="../../../assets/images/notification-bot/notification-new-scenario-diagram.png" alt-text="weather forecast sample notification scenario" lightbox="../../../assets/images/notification-bot/notification-new-scenario-diagram.png":::
+
+You can create notification bot in other scenarios, such as a notification can be sent in teams DevOps channel if there's a build failure.
+
+**Advantages**
+
+* Facilitates notifications to a personal, group chat, and in a channel, using APIs from TeamsFx SDK.
+* Enhances user experience by customizing notification with an Adaptive Card.
+* Provides multiple mechanisms to trigger notifications such as HTTP and schedule timer trigger with Azure Functions.
+
+> [!NOTE]
+> Bot application needs to be installed with the corresponding scope before sending notification.
+
+## Notification based on events
+
+Bot Framework SDK provides the functionality to proactively message in Teams. TeamsFx SDK provides the functionality to manage bot's conversation references when a bot event is triggered. TeamsFx SDK recognizes the following bot events:
+
+|**Event**  |**Behavior**  |
+|---------|---------|
+|The first time you install a bot to a person, group, or Team.     |You need to add the target conversation reference to the storage.         |
```

Comment:
It seems the wordings "*you need to*" are redundant. This section is to describe the underlying tech logic of TeamsFx SDK, and no user action needed.

Reply:
"you need to" has been removed.